Description: no need for shebank in completion script
 A completion script is not a plain script (it is sourced)
Author: Vincent Danjean <vdanjean@debian.org>
Forwarded: yes
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/completion/kameleon.bash
+++ b/completion/kameleon.bash
@@ -1,5 +1,3 @@
-#!/usr/bin/env bash
-
 _kameleon() {
     COMPREPLY=()
     local i=1
